code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -350,11 +350,6 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 		/obj/item/attachable/suppressor = list(CAT_ATT, "Suppressor", 0,"black"),
 		/obj/item/attachable/extended_barrel = list(CAT_ATT, "Extended barrel", 0,"orange"),
 		/obj/item/attachable/compensator = list(CAT_ATT, "Recoil compensator", 0,"black"),
-		/obj/item/attachable/efflens = list(CAT_ATT, "M43 Efficient lens", 0,"black"),
-		/obj/item/attachable/focuslens = list(CAT_ATT, "M43 Focus lens", 0,"black"),
-		/obj/item/attachable/widelens = list(CAT_ATT, "M43 Wide lens", 0,"black"),
-		/obj/item/attachable/heatlens = list(CAT_ATT, "M43 Heat lens", 0,"black"),
-		/obj/item/attachable/pulselens = list(CAT_ATT, "M43 pulse lens", 0,"black"),
```

Comment:
Are those attachments lost? I saw that a bunch of regular ones were re-added. Why's that? Sprite pain?

Reply:
Bloat in the vendor probably.

Reply:
They still exist in code, just with the total replacement of the m43, there isn't any reason to keep them. No (normally) accessible gun can use them.

Reply:
Can't they just be renamed and used for the new Laser Rifle? I mean, laser rifle is laser rifle, regardless of the name and looks, or am I missing something? 
Bloating is an effect too, that's true.

Reply:
They're lens created to do what the current one has standard now, no point to them.